scripts/ec2gaming.bat.template

```diff
@@ -3,6 +3,6 @@ rmdir /Q /S  "C:\Program Files (x86)\Steam\steamapps"
 md Z:\SteamLibrary\steamapps
 cmd /c mklink /j "C:\Program Files (x86)\Steam\steamapps" Z:\SteamLibrary\steamapps
 md Z:\Documents
-aws s3 sync Z:\Documents s3://BUCKET/Documents
+aws s3 sync Z:\Documents s3://BUCKET/Documents --region REGION
 if %errorlevel% neq 0 exit /b %errorlevel%
 schtasks /Create /RU USERNAME /RP PASSWORD /F /SC MINUTE /MO 1 /TN "Sync Documents with S3" /TR "aws s3 sync Z:\Documents s3://ec2gaming-639801188054/Documents --delete"

```

Comment:
Add `--region REGION` to the `aws s3` invocation on line 8 as well, so that it is added to the scheduled task created by this script.